LinuxPBA/UnlockSEDs.cpp

```diff
@@ -35,7 +35,11 @@ uint8_t UnlockSEDs(char * password) {
     LOG(D4) << "Enter UnlockSEDs";
     mvprintw(6,2,"Scanning....");
     while (TRUE) {
-        snprintf(devref,23,"/dev/sd%c",(char) 0x61+i);
+        switch(DtaDevOS::getNextDevice(i)){
+            case 1  : snprintf(devref,23,"/dev/%s",DtaDevOS::getDeviceName()); break;
+            case -1 : snprintf(devref,23,"/dev/sd%c",(char) 0x61+i);    break;
```

Comment:
Since 0x61 was there before I think it is out of scope of this PR to make such refactoring. I suggest to create separate branch for refactoring and replace 0x61 everywhere if needed.